runtime/doc/treesitter.txt

```diff
@@ -422,8 +422,9 @@ instance, to highlight comments differently per language: >vim
     hi @comment.lua guifg=DarkBlue
     hi link @comment.documentation.java String
 <
-The following captures are linked by default to standard |group-name|s (use
-|:Inspect| on a group to see the current link):
+The following is a list of standard captures used in Nvim queries,
```

Comment:
```suggestion
The following is a list of standard captures used in queries for Nvim,
```
Sorry. But that at least is marginally correct, unlike what the spelling police made of my suggestion.

Reply:
Sure. Sounds better.